examples/ndarray/reduce_expr_save2.py

```diff
@@ -1,9 +1,9 @@
 import blosc2
 
 # Create arrays with specific dimensions
-a = blosc2.full((2, 3, 4), 1, urlpath="a.b2nd", mode="w")    # 3D array with dimensions (2, 3, 4)
```

Comment:
If you'd rather keep this as is, I could add `# fmt: off`/`# fmt: on` or `# fmt: skip` pragma comments.